### PR TITLE
Added CITATION.md

### PR DESCRIPTION
**Pull request by divyankgupta23**

### CODE REVIEW

**Review by ashakhatri007 — COMMENTED** [review]
Do we want to change these names to something which is more relatable to our project?

File: CITATION.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cff-version: 1.0.0
+title: TUSK
+abstract: Abstract
+authors:
+  - family-names: Druskat
```
Comment (ashakhatri007) [inline CITATION.md:5]: I guess we should change these names

File: CITATION.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    given-names: Stephan
+    orcid: "https://orcid.org/0000-0003-4925-7248"
+version: 1.0.0
+date-released: "2021-08-29"
+identifiers:
+  - description: First release
+    type: doi
+    value: "https://zenodo.org/badge/latestdoi/401146593"
```
Comment (ashakhatri007) [inline CITATION.md:13]: Is there a way we can see the zenodo badge on the repo?

Comment (divyankgupta23) [reply CITATION.md:13]: We can see zenodo doi in the webhooks section. But for that I have to do the same  process in Neelkanth's repo. Will update you about it.

Comment (divyankgupta23) [reply CITATION.md:13]: Now you'll be able to see zenodo link in the webhook section of this repo.

**Pull request merged**